src/calculateRentalCost.js

```diff
@@ -25,10 +25,24 @@
  * @param {number} days
  *
  * @return {number}
+ *
  */
+const pay = 40;
```

Comment:
You need to initialize this and other variables inside `calculateRentalCost` function.